core/src/main/java/org/opensearch/sql/analysis/ExpressionReferenceOptimizer.java

```diff
@@ -26,25 +25,20 @@
 import org.opensearch.sql.planner.logical.LogicalWindow;
 
 /**
- * The optimizer used to replace the expression referred in the SelectClause
- * e.g. The query SELECT abs(name), sum(age)-avg(age) FROM test GROUP BY abs(name).
- * will be translated the AST
- * Project[abs(age), sub(sum(age), avg(age))
- *  Agg(agg=[sum(age), avg(age)], group=[abs(age)]]
- *   Relation
- * The sum(age) and avg(age) in the Project could be replace by the analyzed reference, the
- * LogicalPlan should be
- * LogicalProject[Ref("abs(age)"), sub(Ref("sum(age)"), Ref("avg(age)"))
- *  LogicalAgg(agg=[sum(age), avg(age)], group=[abs(age)]]
- *   LogicalRelation
+ * The optimizer used to replace the expression referred in the SelectClause e.g. The query SELECT
```

Comment:
This javadoc loses info with merging lines. Probably you need to add javadoc tags (`pre` I suppose) to keep it visual and readable.
Please, go though all changed multiline javadocs to review.